### DISCUSSION TITLE
Bluetooth enumerator is not listed as device and controller cannot connect

### DISCUSSION THREAD

**Discussion by GerroDen** [nefarius/BthPS3#67] [Support]
Hi,

I previously used SCP 1.2.2 and removed everything as far as I could see according to the notes.
Then I installed the new solution and restarted the system after each step.
USB connection with the controller seems to be fine but Bluetooth connection does not work.
The enumerator isn't even listed in the devices and the config tool cannot start.

I am using Window 10 Pro N, 22H2

**Comment by nefarius** [comment]
If the enumerator isn't listed, nothing will ever work. Ever. Reinstall, check the FAQs, [use Legacinator](https://github.com/nefarius/Legacinator).

Cheers

**Reply by GerroDen** [reply]
Thank you very much.
I used the Legacinator to remove an old SCP remnant. This somehow also removed the working normal driver for my working USB Bluetooth 4.0 adapter, but I was able to restore it via the Microsoft update catalogue again with some trial and error 😅.
Reinstallation of the bthps3 driver after that worked and the controller is now connected via Bluetooth. :D
#solved
